internal/controller/restfuncs.go

```diff
@@ -76,7 +78,7 @@ func callbackFunc(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	appErr := handler.CallbackHandler(cbAlert, req.Method)
+	appErr := callback.CallbackHandler(cbAlert, req.Method)
```

Comment:
Why did callback need its own package?

Reply:
we need it to resolve the recursive dependency issue
autoevent uses handler, and callback needs to update autoevent